Makefile

```diff
@@ -32,6 +32,12 @@ install:
 setup:
 	go get -u github.com/golang/dep/cmd/dep
 	go get -u github.com/mjibson/esc/...
+	curl -o assets/prism-bash.js https://raw.githubusercontent.com/PrismJS/prism/v1.16.0/components/prism-bash.js
+	curl -o assets/prism.js https://raw.githubusercontent.com/PrismJS/prism/v1.16.0/prism.js
+	curl -o assets/prism.css https://raw.githubusercontent.com/PrismJS/prism/v1.16.0/themes/prism.css
+	curl -o assets/prism-powershell.js https://raw.githubusercontent.com/PrismJS/prism/v1.16.0/components/prism-powershell.js
+	curl -o assets/materialize.min.css https://raw.githubusercontent.com/Dogfalo/materialize/v1-dev/dist/css/materialize.min.css
+	curl -o assets/materialize.min.js https://raw.githubusercontent.com/Dogfalo/materialize/v1-dev/dist/js/materialize.min.js
```

Comment:
Is there is any concerns not to pre-downloads these js into `assets/` folder?

Reply:
Thanks for review. What do you mean with pre-download? Storing external source code in the git repo could (imho) lead to licensing issues. I do not like this solution too. I tried  to install materialize and prism using dep but coult not get it to work (See golang/dep#1306).

Reply:
ok, thanks.